indico/web/assets/bundles.py

```diff
@@ -240,6 +240,7 @@ def rjs_bundle(name, *files):
     'js/lib/jquery.multiselect.js',
     'js/lib/jquery.multiselect.filter.js',
     'js/jquery/jquery-migrate-silencer.js' if not Config.getInstance().getDebug() else None] +
+    ['js/lib/jquery.tablesorter.js'] +
```

Comment:
why not simply add it to the list before the conditional migrate-silencer entry?